09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>The prose states "20 collision steps", but the code has <code>period=20</code> . Do you mean "timesteps" or <code>period=integrator.collision_method.period * 20</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>This is a typo, steaming is performed every collision (20 timesteps). Use <code>period=integrator.collision_method.period</code> to help clarify this</p>